test/getConfig.spec.js

```diff
@@ -199,56 +208,11 @@ describe('getConfig', () => {
         matchBase: false,
         dot: true
       },
-      linters: {
-        '*.js': 'eslint'
-      },
+      linters: [{ includes: ['*.js'], excludes: [], commands: ['eslint'] }],
```

Comment:
No tests for `excludes`? Let's add some?

Reply:
I haven't yet gotten to tests. Wanted to get the approach validated first. Will do.